app/views/includes/system-messages.njk

```diff
@@ -0,0 +1,32 @@
+{% macro systemMessages(params) %}
```

Comment:
Be good to show a screenshot of this file in the PR.

Reply:
added